packages/react/src/tokens/colors.ts

```diff
@@ -125,7 +125,7 @@ export const colors = {
   "bg.brand.solid.hover": colorPalette["brand.600"],
   "bg.brand.subtle": colorPalette["brand.50"],
   "bg.default.hover": colorPalette["neutral.75"],
-  "bg.disabled": colorPalette["neutral.50"],
+  "bg.disabled": colorPalette["neutral.150"],
```

Comment:
where did you see this change in figma?

Reply:
My bad on this one bhai. I may have seen wrong

Reply:
reverting the changes